lib/gooddata/models/automated_data_distribution.rb

```diff
@@ -6,7 +6,19 @@
 
 module GoodData
   class AutomatedDataDistribution < Rest::Resource
-    attr_accessor :output_stage
+    attr_writer :output_stage
+
+    def output_stage
+      if @output_stage
+        @output_stage
+      else
+        url = project.uri.gsub('/gdc/projects/', '/gdc/dataload/projects/') + '/outputStage'
+        data = project.client.get(url)
+        # TODO: We need to pass ADS but how to get it?
```

Comment:
I don't understand. What is the context? In outputStage, the only way we can get ADS is that outputStage is configured its schemas.